content/blog/2024/01/25/2024-01-25-jenkins-2023-recap.adoc

```diff
@@ -0,0 +1,231 @@
+* The link:/books/[Books] page was updated with new additions & formatting.
+* The link:https://contributors.jenkins.io/[Contributor Spotlight] page was launched to highlight the heaviest contributors to Jenkins. The goal is to appreciate and showcase the talent and hard work that goes into keeping Jenkins working behind the scenes.
+** This was done in collaboration with the Outreach & Advocacy SIG.
+
+There were also several enhancements to Jenkins.io for mobile users so that regardless of platform, everyone can access and read every screen.
+Additionally, there is a new layout for the blog, where each post is displayed as a card.
+
+There was also the addition of UpdateCLI to the jenkins.io repository.
+This has helped ensure that whenever new versions of Jenkins are released, the documentation is updated accordingly.
+Thanks to link:/blog/authors/gounthar/[Bruno Verachten] for his work on getting this configured and added.
+
+The link:/node/tags/gsoc2023/[Google Summer of Code] participants also provided various contributions to both Jenkins core and Jenkins.io, sharing their experiences and insights with the community.
+
+In the coming year, we are also planning on implementing a versioned documentation site, where users would be able to select which Jenkins LTS version they are using and see the corresponding documentation.
+This is the result of a Google Summer of Code project, link:/blog/2023/09/24/building-jenkinsio-with-alternative-tools/[originally looking at alternative build tools for jenkins.io]. Thanks to link:/blog/authors/krisstern/[Kris Stern] and link:/blog/authors/vandit1604/[Vandit Singh] for all of their work on this.
+
+[[outreach]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]
+
+Contributed by: author:alyssat[Alyssa Tong]
+
+In 2023, through the collaboration and contributions of new and existing community members from around the globe, the Jenkins project successfully completed the following projects for the betterment of Jenkins:
+* The launching of a new site, link:https://contributors.jenkins.io/[contributors.jenkins.io], is dedicated to highlighting top Jenkins contributors who are dedicating their time and talent to shape the future of Jenkins.
+* Participated in link:/projects/gsoc/2023/[Google Summer of Code 2023]
+* Welcomed 80+ new contributors with over 400 pull requests merged in link:/blog/2023/09/20/Hacktoberfest-2023/[Hacktoberfest 2023]
+* Participated in link:/blog/2023/09/06/devops-world-tour/[five DevOps World locations, with community speakers]
+** Tim Jacomb - London
+** Olivier Lamy - Singapore
+** Mark Waite - New York, Chicago, and Santa Clara
+
+The Jenkins project is also excited to share what’s to come in 2024:
+* Jenkins in GSoC 2024: link:/blog/2023/12/05/google-summer-of-code-a-call-for-mentors/[Call for Project Ideas + Call for Mentors].
```

Comment:
```suggestion
The Jenkins project is also excited to share what’s to come in 2024:

* Jenkins in GSoC 2024: link:/blog/2023/12/05/google-summer-of-code-a-call-for-mentors/[Call for Project Ideas + Call for Mentors].
```
Same as above, adding break between `:` and first point so that list is formatted accordingly.